tests/sentry/integrations/gitlab/test_issues.py

```diff
@@ -140,3 +140,65 @@ def test_get_issue(self):
                 'display_name': key,
             }
         }
+
+    @responses.activate
+    def test_create_issue_default_project_in_group_api_call(self):
+        group_description = (
+            u'Sentry Issue: [%s](%s)\n\n'
+            '```\nStacktrace (most recent call last):\n\n'
+            '  File "sentry/models/foo.py", line 29, in build_msg\n'
+            '    string_max_length=self.string_max_length)\n\nmessage\n```'
+        ) % (
+            self.group.qualified_short_id,
+            absolute_uri(self.group.get_absolute_url()),
+        )
+        project_id = 10
+        project_name = 'This_is / a_project'
+        self.installation.org_integration.config['project_issue_defaults'] = {
+            self.group.project_id: {'project': project_id}
+        }
+        responses.add(
+            responses.GET,
+            u'https://example.gitlab.com/api/v4/groups/%s/projects' % self.installation.model.name,
```

Comment:
Should this be using the group id instead of the name?

Reply:
@markstory Yes. That is why I brought up the group id concern earlier. Now I can rebase against master and add that in.